webapp/src/pages/tagConfig/components/TagConfigComponent.js

```diff
@@ -86,9 +87,10 @@ class TagConfigComponent extends Component {
     }
   };
 
-  onClickDelete = () => {
-    const tag = null; //TODO
-    tagsService.deleteTag(tag).then(result => { //TODO: will fail, need to add deleteTag to tagsService
+  onClickDelete = (tag) => {
+    //TODO - debug, not working
```

Comment:
This delete is not actually deleting the tag in the DB - it looks okay to me. Am I missing something?